[2025-05-07T15:26:30.962+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-07T15:26:30.989+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.load_task scheduled__2025-05-06T00:00:00+00:00 [queued]>
[2025-05-07T15:26:31.001+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.load_task scheduled__2025-05-06T00:00:00+00:00 [queued]>
[2025-05-07T15:26:31.002+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-05-07T15:26:31.018+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_task> on 2025-05-06 00:00:00+00:00
[2025-05-07T15:26:31.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'load_task', 'scheduled__2025-05-06T00:00:00+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmprm_sowz1']
[2025-05-07T15:26:31.029+0000] {standard_task_runner.py:105} INFO - Job 182: Subtask load_task
[2025-05-07T15:26:31.029+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=176) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-07T15:26:31.030+0000] {standard_task_runner.py:72} INFO - Started process 177 to run task
[2025-05-07T15:26:31.081+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.load_task scheduled__2025-05-06T00:00:00+00:00 [running]> on host e5bea2914fd7
[2025-05-07T15:26:31.178+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-06T00:00:00+00:00'
[2025-05-07T15:26:31.181+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-07T15:26:31.213+0000] {etl_pipeline.py:45} INFO - Données à charger : [['2025-05-07', 9.2, 16.4, 0.0, 3], ['2025-05-08', 9.1, 19.3, 0.0, 3], ['2025-05-09', 9.6, 19.9, 0.0, 3], ['2025-05-10', 9.6, 22.2, 0.0, 1], ['2025-05-11', 11.8, 26.2, 0.0, 2], ['2025-05-12', 14.9, 21.6, 11.1, 80], ['2025-05-13', 13.1, 21.2, 7.2, 95]]
[2025-05-07T15:26:31.228+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-07T15:26:31.238+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-07T15:26:31.239+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=load_task, run_id=scheduled__2025-05-06T00:00:00+00:00, execution_date=20250506T000000, start_date=20250507T152630, end_date=20250507T152631
[2025-05-07T15:26:31.286+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-07T15:26:31.304+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-07T15:26:31.306+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
